recipes/libarchive/all/conanfile.py

```diff
@@ -93,25 +93,25 @@ def requirements(self):
         if self.options.with_iconv:
             self.requires("libiconv/1.17")
         if self.options.with_pcreposix:
-            self.requires("pcre2/10.42")
+            self.requires("pcre/8.45")
         if self.options.with_nettle:
             self.requires("nettle/3.9.1")
         if self.options.with_openssl:
             self.requires("openssl/[>=1.1 <4]")
         if self.options.with_libb2:
             self.requires("libb2/20190723")
         if self.options.with_lz4:
-            self.requires("lz4/1.9.4")
+            self.requires("lz4/1.10.0")
```

Comment:
```suggestion
            self.requires("lz4/1.9.4")
```

It would conflict elsewhere, let's keep this for now, maybe a good version range target for the future?